vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // C wrapping code
 
-// #cgo LDFLAGS: -lseccomp
+// #cgo pkg-config: libseccomp
```

Comment:
> But since 40b21745cc14628fc5336fbacddf092667a70c65 it uses "gcc 6.1".

When I use gcc 6.1 (go 1.6.1) also on s390x the problem disappears.
I created PR #23108 for using gcc 6.1 also for s390x.